### PR TITLE
finished with readmemd

### PR DESCRIPTION
**Pull request by himanshujasuja1040**
@stephin007

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,47 @@
 
 ---
 
-> TechStack - React JS, Material UI
+## Installation
 
-_MORE INFO YET TO BE ADDED... STAY TUNED_
+The Open Event Frontend can be easily deployed on a variety of platforms, so just run the following commands when setting up for the first time:
```
Comment (stephin007) [inline README.md:7]: deployed on various platforms?

This project is not redeployable, Please change this text.

File: README.md [hunk 2]
```diff
@@ -2,12 +2,47 @@
...
 
-# DON'T JUST CLONE, PLEASE LEAVE A STAR, MOTIVATES US TO ADD MORE FEATURES TO THE PROJECT.
+- `yarn install or npm install  `
+- `yarn start or npm start `
+- Visit your app at [http://localhost:3000](http://localhost:3000).
 
-> NEW UI COMING UP , CHECK THIS SITE TO SEE THE LATEST UPDATES : [https://cowinvaccinetracker.stephinreji.me/](https://cowinvaccinetracker.stephinreji.me/)
+CoWIN Vaccination Slots Checking App is a website, mobile-ready,
+REACT JS-along with Material-UI
```
Comment (stephin007) [inline README.md:14]: Line 14: Does Not make any sense here, since you have added the same under tech header in line 29

File: README.md [hunk 3]
```diff
@@ -2,12 +2,47 @@
...
 
-### _NOTE:_ Please check the Projects section to see the progress and the discussion panel to add your ideas, which will eventually be picked up for development
+## Features
+
+CoWIN Vaccination Slots Checking App is a user Friendly website
+that allow users to find vaccine in nearby available Center.
+
+> The overriding design goal for CoWIN is
+> helping people around nation to
+> book their slot by providing details of
```
Comment (stephin007) [inline README.md:23]: We don't help in booking slots, we can only view the slots.

Please change the text here

File: README.md [hunk 4]
```diff
@@ -2,12 +2,47 @@
...
+> available vaccine at particular Center
+> publishable as-is, as plain text, without
+> looking like it's been marked up with tags
+> or formatting instructions.
+
```
Comment (stephin007) [inline README.md:28]: What does line 25 - 27 mean?

File: README.md [hunk 5]
```diff
@@ -2,12 +2,47 @@
...
+---
+
+> _DON'T JUST CLONE, PLEASE LEAVE A STAR, MOTIVATES US TO ADD MORE FEATURES TO THE PROJECT_
+
+---
+
+#### NEW UI COMING UP , CHECK THIS SITE TO SEE THE LATEST UPDATES :
+
+##### https://cowinvaccinetracker.stephinreji.me/
```
Comment (stephin007) [inline README.md:44]: Please add the rest of the demo links.

File: README.md [hunk 6]
```diff
@@ -2,12 +2,47 @@
...
-> TechStack - React JS, Material UI
+## Installation
 
-_MORE INFO YET TO BE ADDED... STAY TUNED_
+The Open Event Frontend can be easily deployed on a variety of platforms, so just run the following commands when setting up for the first time:
 
-# DON'T JUST CLONE, PLEASE LEAVE A STAR, MOTIVATES US TO ADD MORE FEATURES TO THE PROJECT.
+- `yarn install or npm install  `
+- `yarn start or npm start `
+- Visit your app at [http://localhost:3000](http://localhost:3000).
```
Comment (stephin007) [inline README.md:11]: After this line , ask the user to fetch the NEW-UI branch from the origin and the create a branch from it, as staging and master are protected branches, All PRs to Staging/Master will not be considered.

**Review by stephin007 — CHANGES_REQUESTED** [review]
Please resolve the comments , thanks!

**Review by Justinnn07 — APPROVED** [review]
LGTM!

**Review by stephin007 — APPROVED** [review]

**Pull request merged**